content/en/tracing/configure_data_security/_index.md

```diff
@@ -13,31 +13,129 @@ aliases:
+| Low        | Data that is available publicly or contains non-identifying system information                                       | Geographic regions, host ports                                |
 
-* **Environment variables are not collected by the Agent**
-* **SQL variables are obfuscated, even when not using prepared statements**: For example, the following `sql.query` attribute: `SELECT data FROM table WHERE key=123 LIMIT 10` has its variables obfuscated, to become the following Resource name: `SELECT data FROM table WHERE key = ? LIMIT ?`
-* **SQL strings are identified using standard ANSI SQL quotes**: This means strings should be surrounded in single quotes (`'`). Some SQL variants optionally support double-quotes (`"`) for strings, but most treat double-quoted things as identifiers. The Datadog obfuscator treats these as identifiers rather than strings and does not obfuscate them.
-* **Numbers in Resource names (for example, request URLs) are obfuscated** For example, the following `elasticsearch` attribute:
+Values attributed to these categories are collected differently dependending on the language library. Generally, the following are considered higher-risk categories:
 
-    ```text
-    Elasticsearch : {
-        method : GET,
-        url : /user.0123456789/friends/_count
-    }
-    ```
+| Data category       | Description                                                                                                                               |
+|:--------------------|:------------------------------------------------------------------------------------------------------------------------------------------|
+| Name                | The real name of an internal user (your employee) or end-user.                                                                                 |
+| Email               | The email address of an internal user (employee) or end-user.                                                                             |
+| Client IP           | The IP address of your end-user associated with an incoming request or the external IP address of an outgoing request.                    |
+| Database statements | The literal, sequence of literals, or bind variables used in an executed database statement.                                              |
+| Geographic location | Longitude and latitude coordinates that can be used to identify an individual or household.                                               |
+| URI parameters      | The parameter values in the variable part of the URI path or the URI query.                                                               |
+| URI userinfo        | The userinfo subcomponent of the URI that may contain the user name.                                                                      |
+
+These categories are collected by the following language Tracing Libraries:
```

Comment:
We should clarify that collection pertains to the automatic instrumentation provided by each library.